webview/src/experiments/components/table/styles.module.scss

```diff
@@ -150,18 +147,11 @@ $workspace-row-edge-margin: $edge-padding - $cell-padding;
       animation: spin 1s cubic-bezier(0.53, 0.21, 0.29, 0.67) infinite;
       background-color: $bg-color;
     }
-    .runningExperiment.oddRow & {
-      background-color: $row-bg-alt-color;
-    }
+
     .workspaceWithChanges.runningExperiment & {
       border-right-color: $changed-color;
       border-top-color: $changed-color;
     }
-
-    .runningExperiment.oddRow > .experimentCell & {
-      border-left: 1.5px solid $row-bg-alt-color;
-      border-bottom: 1.5px solid $row-bg-alt-color;
-    }
```

Comment:
[I] Couple of cases to follow up on:

1. Middle of an unselected experiment's circle whilst the row is highlighted

<img width="1728" alt="Screen Shot 2022-08-16 at 9 33 44 am" src="https://user-images.githubusercontent.com/37993418/184745633-c0584ac6-44e0-4e88-a95b-d6e58d584dc6.png">

2. Middle of a running experiment's spinner whilst highlighted

<img width="1728" alt="Screen Shot 2022-08-16 at 9 32 42 am" src="https://user-images.githubusercontent.com/37993418/184746360-b91ff0d3-bc01-4e53-a77c-1cedd301eb3b.png">